ide/xsl/src/org/netbeans/modules/xsl/grammar/XSLGrammarQuery.java

```diff
@@ -365,7 +365,7 @@ private static Set getTemplate() {
 
     private static Set getXslFunctions() {
         if (xslFunctions == null) {
-            xslFunctions = new TreeSet(Arrays.asList(new String[]{
+            xslFunctions = new TreeSet<>(Arrays.<String>asList(new String[]{
```

Comment:
inferrable type informations
```suggestion
            xslFunctions = new TreeSet<>(Arrays.asList(new String[]{
```
strictly speaking the array creation is not needed as well

Reply:
changed..